### ISSUE TITLE
Add timeout to sending commands

### ISSUE THREAD

**Issue by onik** [Kemmey/Kemmey-TeslaWatch-Public#329]
Hello,

This morning on my way out the door I gave a command to start the AC. The car was sleeping so the command didn't go through before I got to the garage and in my car.

However, when I got to my office and raised my wrist, I heard a familiar pling and the command went through.

Please add at least an option to give a timeout to trying to send a command to avoid having to monitor whether the command goes through or not.

**Comment by Kemmey** [comment]
@onik yeah - that's definitely something I'm aware of and am actively trying to fix. There's two parts to this - one is if the request fails during wake, and the other is if networking fails during command execution. 

Wake should be fixed in latest app store release - but if network issues occur during actual command execution, the OS will keep retrying the request once network connectivity is re-established, even though the app has long since cancelled the request. At least that's what I'm experiencing and is by definition the bug. I've tried a lot of things to get around this, but so far, nothing has stuck. 

Please do keep me posted if this is something you can actively reproduce - I'd like as much info about under which circumstances this happens as possible, so I can add those to my test suite. 

Cheers, 

Kim
